src/rgw/rgw_sal_motr.cc

```diff
@@ -3609,16 +3614,16 @@ void *newMotrStore(CephContext *cct)
     const auto& admin_proc_ep  = g_conf().get_val<std::string>("motr_admin_endpoint");
     const auto& admin_proc_fid = g_conf().get_val<std::string>("motr_admin_fid");
     const int init_flags = cct->get_init_flags();
-    ldout(cct, 0) << "INFO: motr my endpoint: " << proc_ep << dendl;
-    ldout(cct, 0) << "INFO: motr ha endpoint: " << ha_ep << dendl;
-    ldout(cct, 0) << "INFO: motr my fid:      " << proc_fid << dendl;
-    ldout(cct, 0) << "INFO: motr profile fid: " << profile << dendl;
+    ldout(cct, 20) << "INFO: motr my endpoint: " << proc_ep << dendl;
+    ldout(cct, 20) << "INFO: motr ha endpoint: " << ha_ep << dendl;
+    ldout(cct, 20) << "INFO: motr my fid:      " << proc_fid << dendl;
+    ldout(cct, 20) << "INFO: motr profile fid: " << profile << dendl;
```

Comment:
This was a recommendation from QA that these logs should not print by default, so I have increased the log level here. Let me know if this is okay, otherwise it will reverse this change. @andriytk @bkirunge7

Reply:
@jjxsg I agree with @andriytk . Lets keep these log levels as is.

Reply:
Hi, If I am running `radosgw-admin user info` cmd I get 10 lines of logs first before the actual user info, does a user need that? if the cmd is failing he can run the same cmd with `--debug-rgw=[0-99]` and get the logs anyway.
@andriytk , @sachinpunadikar  ??

Reply:
The point from QA was that there should be a way to enable/disable these logs.

Reply:
@jjxsg & @sumedhak27 , your points are valid. But till the time automated deployment of rgw is stabilizes, let have these entries with debug level 0. In next PI, we will convert with debug level of say 50, when we have sufficient confidence that our deployments steps are stable.

Reply:
Oh, I didn't know that `radosgw-admin` prints all these on every run... Is there any way to switch it off? I see from the sample above that it prints not only 0 level, but level 1 also:

```
2022-02-17T21:49:23.778-0700 7f2edc5c25c0  1 RGW CONF BACKEND STORE = motr
2022-02-17T21:49:23.778-0700 7f2edc5c25c0  1 RGW BACKEND STORE = motr
2022-02-17T21:49:23.778-0700 7f2edc5c25c0  0 INFO: motr my endpoint: 192.168.180.182@tcp:12345:4:1
...
```

Reply:
Default log level is 20, that's why all these logs are getting printed.